src/backend/distributed/executor/multi_utility.c

```diff
@@ -850,6 +863,30 @@ ErrorIfUnsupportedAlterTableStmt(AlterTableStmt *alterTableStatement)
 		{
 			case AT_AddColumn:
 			{
+				if (IsA(command->def, ColumnDef))
+				{
+					ColumnDef *column = (ColumnDef *) command->def;
+
+					/* Check for SERIAL pseudo-types */
+					if (column->typeName && list_length(column->typeName->names) == 1 &&
```

Comment:
I hope so, since this is copy-pasted-edited from `transformColumnDefinition`. I'll add a comment.